lib/ts/recipe/emailpassword/components/themes/default/emailVerification/emailVerificationScreen/verifyEmailLinkClicked.tsx

```diff
@@ -0,0 +1,138 @@
+/* Copyright (c) 2020, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*
+ * Imports.
+ */
+import React, { PureComponent } from "react";
+import StyleContext from "../../../../styles/styleContext";
+
+/** @jsx jsx */
+import { jsx } from "@emotion/react";
+import { VerifyEmailLinkClickedThemeProps, VerifyEmailLinkClickedThemeState } from "../../../../../types";
+import SpinnerIcon from "../../../../../assets/spinnerIcon";
+import { VERIFY_EMAIL_LINK_CLICKED_STATUS } from "../../../../../constants";
+import ErrorLargeIcon from "../../../../../assets/errorLargeIcon";
+import ArrowRightIcon from "../../../../../assets/arrowRightIcon";
+import { Button } from "../../../../library";
+import CheckedRoundIcon from "../../../../../assets/checkedRoundIcon";
+
+/*
+ * Component.
+ */
+
+export default class VerifyEmailLinkClickedTheme extends PureComponent<
+    VerifyEmailLinkClickedThemeProps,
+    VerifyEmailLinkClickedThemeState
+> {
+    static contextType = StyleContext;
+
+    /*
+     * Constructor.
+     */
+    constructor(props: VerifyEmailLinkClickedThemeProps) {
+        super(props);
+        this.state = {
+            status: VERIFY_EMAIL_LINK_CLICKED_STATUS.LOADING
+        };
+    }
+
+    onSuccess = (): void => {
+        this.setState(() => ({
+            status: VERIFY_EMAIL_LINK_CLICKED_STATUS.SUCCESSFUL
+        }));
+
+        if (this.props.onSuccess !== undefined) {
+            this.props.onSuccess();
+        }
+    };
+
+    componentDidMount = async (): Promise<void> => {
+        const response = await this.props.callAPI();
```

Comment:
How are 500 errors from the API handled?

Reply:
They are handled in `emailVerification/api.ts`. No need to handle here. 

`sendVerifyEmail` was handling it but it was useless so I'm cleaning.